lib/utils/users.ts

```diff
@@ -0,0 +1,32 @@
+import cookie from 'cookie';
+import jsonwebtoken from 'jsonwebtoken';
+import { HackneyResident } from '../../domain/HackneyResident';
+
```

Comment:
I feel like we need a clearer separation than this `utils` structure is allowing.
The `getUser` function appears to only execute server side - it's called from the getServerSideProps function and it won't work in the browser.
On the other hand - `signOut` is running in the browser. I think the problem is that these functions both have to be executed in their respective environments - `getUser` accesses an env var and `signOut` uses the browser's `window` object.

Reply:
Also - and maybe the build processes have improved these days - but in the past this kind of thing would cause all the `imports` to appear in the client side build and make your JS files big.

Reply:
Yep understood, have updated the functions to all be server side.